### PR TITLE
code clean up: rosa classic scripts

### PR DESCRIPTION
**Pull request by mukrishn**
## Type of change

- [x] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Cleaned up rosa script and moved hosted-cp pieces to a different file.

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: dags/openshift_nightlies/util/manifest.py [hunk 1]
```diff
@@ -248,8 +275,10 @@ def get_releases(self):
             self.get_openstack_releases()
         if 'rosa' in self.yaml['platforms']:
             self.get_rosa_releases()
+        if 'rosahcp' in self.yaml['platforms']:
+            self.get_rosahcp_releases()
         if 'rogcp' in self.yaml['platforms']:
-            self.get_rogcp_releases()
+            self.get_rogcp_releases()
```
Comment (krishvoor) [inline dags/openshift_nightlies/util/manifest.py:281]: Very minor nits on spacing :)

**Review by krishvoor — APPROVED** [review]
/lgtm

File: dags/openshift_nightlies/config/benchmarks/hcp-small-control-plane.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "benchmarks": [
+        {
+            "name": "node-density",
+            "workload": "kube-burner-ocp-wrapper",
+            "trigger_rule": "all_done",
+            "command": "./run.sh",
+            "env": {
+                "WORKLOAD": "node-density",
+                "LOG_LEVEL": "debug",
+                "CHURN": "false"
```
Comment (rsevilla87) [inline dags/openshift_nightlies/config/benchmarks/hcp-small-control-plane.json:11]: Not actually an issue, but...
CHURN is only available on cluster-density workloads at the moment

File: dags/openshift_nightlies/config/benchmarks/hcp-small-control-plane.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        {
+            "name": "cluster-density-v2",
+            "workload": "kube-burner-ocp-wrapper",
+            "trigger_rule": "all_done",
+            "command": "./run.sh",
+            "env": {
+                "WORKLOAD": "cluster-density-v2",
+                "ITERATIONS": "500",
+                "LOG_LEVEL": "debug",
+                "CHURN": "false"
```
Comment (rsevilla87) [inline dags/openshift_nightlies/config/benchmarks/hcp-small-control-plane.json:34]: false?

File: dags/openshift_nightlies/config/benchmarks/hosted-control-plane-p75.json [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
-            "name": "cluster-density-ms-p75",
+            "name": "cluster-density-v2-p75",
             "workload": "kube-burner-ocp-wrapper",
             "trigger_rule": "all_done",
             "command": "./run.sh",
             "env": {
-                "WORKLOAD": "cluster-density-ms",
-                "ITERATIONS": "75",
+                "WORKLOAD": "cluster-density-v2",
+                "ITERATIONS": "63",
```
Comment (dry923) [inline dags/openshift_nightlies/config/benchmarks/hosted-control-plane-p75.json:10]: is 63 iterations the new 75th percentile number roughly? if not we should likely change the name to p63 or something to match

File: dags/openshift_nightlies/config/benchmarks/hosted-control-plane-p90.json [hunk 5]
```diff
@@ -1,22 +1,15 @@
...
-                "HYPERSHIFT": "true",
-                "METRICS_PROFILE": "metrics-profiles/hypershift-metrics.yaml",
-                "QPS": "20",
-                "BURST": "20",
-                "LOG_LEVEL": "info",
-                "PLATFORM_ALERTS": "false",
-                "CLEANUP_WHEN_FINISH": "true",
-                "CLEANUP": "true"
+                "WORKLOAD": "cluster-density-v2",
+                "ITERATIONS": "84",
```
Comment (dry923) [inline dags/openshift_nightlies/config/benchmarks/hosted-control-plane-p90.json:10]: same iteration comment as the p75 workload one

**Review by dry923 — APPROVED** [review]
LGTM!

**Pull request merged**